core/backend/src/RpcBackend.ts

```diff
@@ -63,76 +21,14 @@ export function initializeRpcBackend(enableOpenTelemetry: boolean = false) {
 
   initialized = true;
```

Comment:
should this be set last?

Reply:
Good catch. Yes, it definitely should be...